packages/cashier/src/stores/__tests__/general-store.spec.ts

```diff
@@ -119,38 +119,24 @@ describe('GeneralStore', () => {
     //     expect(general_store.is_p2p_enabled).toBeTruthy();
     // });
 
-    it('should not show p2p in cashier onboarding if p2p_advertiser_error is equal to "RestrictedCountry"', () => {
-        general_store.setP2pAdvertiserError('RestrictedCountry');
-        general_store.showP2pInCashierOnboarding();
+    // it('should show p2p in cashier onboarding if the user account is not virtual, and he has USD account', async () => {
+    //     await general_store.showP2pInCashierOnboarding();
 
-        expect(general_store.show_p2p_in_cashier_onboarding).toBeFalsy();
-    });
-
-    it('should not show p2p in cashier onboarding if the user has accounts with fiat currency, but has not account with USD currency', () => {
-        general_store.root_store.client.account_list = [{ title: 'EUR' }];
-        general_store.showP2pInCashierOnboarding();
-
-        expect(general_store.show_p2p_in_cashier_onboarding).toBeFalsy();
-    });
+    //     expect(general_store.show_p2p_in_cashier_onboarding).toBeTruthy();
+    // });
```

Comment:
Why? 😅

Reply:
oh wait i forgot to uncomment one of the tests, let me check again